ukvm/ukvm-core.c

```diff
@@ -479,7 +479,7 @@ static void setup_cpuid(int kvm, int vcpufd)
         err(1, "KVM_SET_CPUID2 failed");
 }
 
-
+#if 0
```

Comment:
Yeap, agree: let's have a another one just for that instead (cleanup), which doesn't have to be now.